Auth service

amalrajkp18 opened the pull request:
Created Authentication functionalities and implemented to UI

amalrajkp18 committed:
Authentication functionalities created

-created folder structure
-created authentication service
-created authentication use_cases
-created authentication provider

amalrajkp18 committed:
Implemented authentication functionalities in UI

-Edited alert box : text errors,added phone number text section
-Edited constants : remove unwanted text constants
-Implemented providers in UI

Sireen-Official commented on lib/features/authentication/controller/authentication_provider.dart:
Loading toggle missing in some part of code

amalrajkp18 committed:
loading toggle added missing parts

amalrajkp18 committed:
Added: pages

- Changed loading toggle in authentication provider
- added splash page
- changed main page

amalrajkp18 committed:
Core : constants changed

-login page constants name changes

amalrajkp18 committed:
confilcts resolved and minor bugs fixed
